chalice/policies.json

```diff
@@ -326,6 +326,23 @@
     "UnassignPrivateIpAddresses": "ec2:UnassignPrivateIpAddresses",
     "UnmonitorInstances": "ec2:UnmonitorInstances"
   },
+  "es": {
+    "ESHttpDelete": "es:ESHttpDelete",
```

Comment:
How are these used?  The ES* keys don't map to boto3 operations so they'll never get triggered during policy generation.  Can we remove these?